pkg/loki/loki.go

```diff
@@ -399,6 +399,8 @@ func (t *Loki) ListTargets() {
 
 // Run starts Loki running, and blocks until a Loki stops.
 func (t *Loki) Run(opts RunOpts) error {
```

Comment:
If we want a more precise startup time, including some config parsing that happens prior to calling `Run()` we could start the timer in the `main()` function and pass it as a `RunOpt`.
Not sure if it really makes much difference, though.

Reply:
Actually, let me do that!